.github/CODEOWNERS

```diff
@@ -61,7 +61,7 @@
 /examples/field_formats_example/ @elastic/kibana-app-services
```

Comment:
These should go to GX:

- `ui_action_examples`
- `ui_actions_explorer`
-  `state_containers_examples`
- `ui_actions`
- `ui_actions_enhanced`
- `screenshot_mode`
- `bfetch`
- `ui_actions_enhanced_examples`
- `notifications`